### ISSUE TITLE
looper summarize html report loses order

### ISSUE THREAD

**Issue by nsheff** [pepkit/looper#47]
The summary table `tsv` output is ordered correctly, but the HTML output does not follow that column order.

**Comment by nsheff** [comment]
metadata columns are ordered correctly, but stats columns are not.

**Comment by nsheff** [comment]
right now the HTML table builder appears to be taking the data individually from the `stats.tsv` file, instead of from the already created summary table. this is redoing all the work that was already done by the original summarizer.

Instead, this shouldn't re-do that work with the stats (which also re-introduces problems like the above, which have already been solved by the summarizer earlier), but should rely on that code.

In other words, don't read in individual samples and put together a table, build the HTML table from the already existing summary table.

**Comment by nsheff** [comment]
@jpsmith5 can you take a look at this one?

**Comment by jpsmith5** [comment]
Give it a try now.  Should be correctly ordered both on main page and individual sample pages.

**Comment by nsheff** [comment]
yes, the order now appears correct.
